xml2rfc/writers/pdf.py

```diff
@@ -166,6 +174,9 @@ def flatten_unicode_spans(self, html):
     font-family: {fonts};
     width: 100%;
   }}
+  tt, code, pre, code {{
```

Comment:
Reason for having `code` in here twice?

Reply:
Good spotting! It's redundant.
Same in https://github.com/ietf-tools/xml2rfc/blob/main/xml2rfc/data/xml2rfc.css#L790

Reply:
Fixed with #916.